library/Icingadb/Widget/Detail/ObjectDetail.php

```diff
@@ -248,7 +256,6 @@ protected function createNotes()
         if (empty($content)) {
             return null;
         }
-
```

Comment:
Why did you remove this empty line? We require explicit empty lines after control structures.